### PR TITLE
Feature: Add env.isEdge property.

### PR DESCRIPTION
**Pull request by jodator**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Introduce `env.isEdge` property.

---

### Additional information

* Required for:  https://github.com/ckeditor/ckeditor5-widget/pull/46

### CODE REVIEW

File: src/env.js [hunk 1]
```diff
@@ -37,3 +45,13 @@ export default env;
...
 }
+
+/**
+ * Checks if User Agent represented by the string is Edge browser.
+ *
+ * @param {String} userAgent **Lowercase** `navigator.userAgent` string.
+ * @returns {Boolean} Whether User Agent is Edge or not.
+ */
+export function isEdge( userAgent ) {
+	return userAgent.indexOf( 'edge' ) > -1;
```
Comment (Reinmar) [inline src/env.js]: That's unsafe. There are many words with "edge" in them. Something more precise would be better.

Comment (Reinmar) [reply src/env.js]: BTW, feel free to look into CKE4's code:

https://github.com/ckeditor/ckeditor-dev/blob/master/core/env.js#L20

Comment (jodator) [reply src/env.js]: @Reinmar I'm not 100% sure but it looks like `[ /]` in that regex could be restricted to:

```js
/edge\/(\d+.?\d*)/
```

I couldn't find why there is space allowed...

Comment (jodator) [reply src/env.js]: ps.: https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/hh869301(v=vs.85)

Describes only `Edge/<EdgeHTML Rev>.<Windows Build>` as possible value so I've set the regex accordingly to this.

File: tests/env.js [hunk 2]
```diff
@@ -31,4 +37,19 @@ describe( 'Env', () => {
 			expect( isMac( 'foo' ) ).to.be.false;
 		} );
 	} );
+
+	describe( 'isEdge()', () => {
+		it( 'returns true for Edge UA strings', () => {
```
Comment (Reinmar) [inline tests/env.js]: Not a single real UA string ;)

Comment (jodator) [reply tests/env.js]: Yeah - I've copied from other test. I'm adding some real ones right now ;)

Comment (jodator) [reply tests/env.js]: OK... I was too slow :disappointed:

**Pull request merged**